tests/unittests/Foundation/NSFileManagerTests.mm

```diff
@@ -136,4 +170,76 @@
     NSString* currentPath = [manager currentDirectoryPath];
     ASSERT_OBJCNE_MSG(originalPath, currentPath, "Expected change in current directory");
     ASSERT_OBJCEQ_MSG(parentPath, currentPath, "Expected current directory to change to parentPath");
-}
\ No newline at end of file
+}
+
+TEST(NSFileManager, MoveFileViaPath) {
+    char const* srcName = "\\NSFileManagerMoveTestFile.txt";
+    char const* destName = "\\MovedFile.txt";
+
+    NSString* content = @"The Quick Brown Fox.";
+    createFileWithContentAndVerify(srcName, content);
+
+    NSString* srcPath = getPathToFile(srcName);
+    NSString* destPath = getPathToFile(destName);
+
+    NSFileManager* manager = [NSFileManager defaultManager];
+
+    NSError* error = nil;
+    BOOL status = [manager moveItemAtPath:srcPath toPath:destPath error:&error];
+    EXPECT_TRUE(status);
+    EXPECT_EQ(nil, error);
```

Comment:
This Expect nil, error really doesn't do a whole lot. It might be worthwhile to add a test for actual errors instead of a test that just makes sure nil comes back as nil through a function.